Documentation/En/Index.xyl

```diff
@@ -26,28 +26,28 @@
   abbreviated SUT, and is composed of two parts:</p>
   <ol>
     <li><strong>Test data</strong> to execute the SUT, and</li>
-    <li>An <strong>oracle</strong>, establishing the <strong>test
+    <li>an <strong>oracle</strong>, establishing the <strong>test
     verdict</strong>: Are the result of the execution and the state of
-    the SUT after its execution the ones we expect or not?</li>
+    the SUT after it's execution the ones we expect or not?</li>
   </ol>
   <p>The values of the verdict are: <strong>Success</strong>,
   <strong>fail</strong> or <strong>inconclusive</strong>. A success means this
   is what we expect, a fail means this is not what we expect and an
-  inconclusive means this is not possible to give an answer. The important
+  inconclusive means it is not possible to give an answer. The important
```

Comment:
Sorry, I didn't explain correctly. Basically the sentance does not read correctly to me in its current form. I would change it to `means that it is not ...`. Sorry I can't explain better.